src/ipa_pc/mod.rs

```diff
@@ -842,9 +799,8 @@ impl<G: AffineCurve, D: Digest> PolynomialCommitment<G::ScalarField> for InnerPr
         combined_commitment = combined_commitment_proj.into_affine();
 
         // ith challenge
-        let mut round_challenge = Self::compute_random_oracle_challenge(
-            &to_bytes![combined_commitment, point, combined_v].unwrap(),
-        );
+        fs_rng.absorb(&to_bytes![combined_commitment, point, combined_v].unwrap());
```

Comment:
Let's change the comment to "0-th challenge"

Reply:
Done